ansibleRunnerJobEvent.yml

```diff
@@ -0,0 +1,53 @@
+# This schema captures the shape of events produced by ansible runner
+# https://ansible-runner.readthedocs.io/en/stable/intro.html#runner-artifact-job-events-host-and-playbook-events
+# This schema does not aim to be complete - it only captures used by playbook-dispatcher
+---
+$id: ansibleRunnerJobEvent
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  event:
+    type: string
+    minLength: 3
+    maxLength: 50
+  uuid:
+    type: string
+    format: uuid
+  counter:
+    type: integer
+  stdout:
+    type:
+      - string
+      - "null"
+  start_line:
+    type: integer
+    minimum: 0
+  end_line:
+    type: integer
+    minimum: 0
+  event_data:
+    type: object
+    properties:
+      playbook:
+        type: string
+        minLength: 1
+      playbook_uuid:
+        type: string
+        format: uuid
+      host:
+        type: string
+
+      crc_dispatcher_correlation_id:
+        type: string
+        format: uuid
+      crc_dispatcher_error_code:
+        type: string
+      crc_dispatcher_error_details:
+        type: string
```

Comment:
There is nothing marked as required in the `event_data`. Is it purpose?

Reply:
This schema was defined a long time ago by playbook dispatcher, so I cannot explain why none of these properties are required.